scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Deploying a Daemon environment to a cluster with Replace After Terminate
```

Comment:
we should add a TODO: add update env test cases

Reply:
Sure.